python/cudf/groupby/groupby.py

```diff
@@ -139,6 +139,8 @@ def _apply_agg(self, agg_type, result, add_col_values,
         need_to_index = self._as_index
 
         col_count = 0
+        if isinstance(val_columns, str):
```

Comment:
What if the column is an integer?

Reply:
That sounds really important!

Reply:
Hmm I'm not sure we ever supported integer column names for groupby. Looking into it.

Reply:
Fixed, ready for merge.